p2p/config.go

```diff
@@ -16,13 +16,13 @@ type Config struct {
 	// Peers used for bootstrapping
 	BootstrapNodes []string
 	// Listening port
-	Port           int
+	Port int
```

Comment:
why is the formatting removed :'( lol

Reply:
There was an issue with the linter running in travis for the past week that David and I found. It was passing without running the linter due to pulling in an unstable branch. I included the fix in this PR and then proceeded to lint all files that the linter was complaining about.